docs/core/tutorials/using-on-macos.md

```diff
@@ -82,39 +61,27 @@ contains the following information:
 </Project>
 ```
 
-This library project will make use of JSON representation of objects, so you'll want to
-add a reference to the `Newtonsoft.Json` NuGet package. The `dotnet add` command
-adds new items to a project. To add a reference to a NuGet package, you use the 
-`package` command and specify the name of the package. 
+This library project will represent objects in JSON, so add a reference to the `Newtonsoft.Json` NuGet package. The `dotnet add` command adds new items to a project. To add a reference to a NuGet package, use the [`dotnet add package`](../tools/dotnet-add-package.md) command and specify the name of the package:
```

Comment:
"This library project will represent" seems confusing to me -- a project is a collection of source code, and the binaries produced from it perform some operation. Here, we're focused on the operation. So, "Our library methods will serialize and deserialize objects in JSON format. To support JSON serialization and deserialization, add a reference..."